src/Entity/CustomerBillingData.php

```diff
@@ -46,80 +46,108 @@ class CustomerBillingData implements CustomerBillingDataInterface
     /** @var string|null */
     protected $provinceCode;
 
-    public function __construct(
-        string $firstName,
-        string $lastName,
-        string $street,
-        string $postcode,
-        string $countryCode,
-        string $city,
-        ?string $company = null,
-        ?string $provinceName = null,
-        ?string $provinceCode = null
-    ) {
-        $this->firstName = $firstName;
-        $this->lastName = $lastName;
-        $this->street = $street;
-        $this->postcode = $postcode;
-        $this->countryCode = $countryCode;
-        $this->city = $city;
-        $this->company = $company;
-        $this->provinceName = $provinceName;
-        $this->provinceCode = $provinceCode;
+    public function getId(): ?int
```

Comment:
I would be for not forcing the type of identifier
```suggestion
    public function getId()
```